### PR TITLE
show tickets order in descending order

### LINKED ISSUE #3125: Tickets and Orders: Show latest orders first
In the event orders and tickets tab (e.g. `events/event-id/tickets/orders/`) and admin sales tab (`/admin/sales/events/`) please sort to show new order/tickets/events on top and older orders/tickets/events on the bottom.

screenshot from 2017-02-02 08-49-46

### PR DESCRIPTION
**Pull request by poush**
resolves #3125 

Show latest orders first.

screen shot 2017-02-09 at 1 22 44 pm

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -18,6 +18,7 @@ app.db
 
 # Static files at 'app/static/vendor'
 app/static/vendor
+app/static/admin
```
Comment (niranjan94) [inline .gitignore]: why is this line necessary ?

Comment (poush) [inline .gitignore]: Because git was showing this under untracked files. I feel this is generated under installation process and not need to present under git repo.

Comment (niranjan94) [inline .gitignore]: Nope. You can remove this line and delete that folder (`app/static/admin`).

Comment (niranjan94) [inline .gitignore]: That directory was renamed to `app/static/vendor`

Comment (poush) [inline .gitignore]: Done

File: app/helpers/ticketing.py [hunk 2]
```diff
@@ -71,6 +71,7 @@ def get_orders(event_id=None, status=None, from_date=None, to_date=None, markete
 
             orders = orders.order_by(asc(Order.created_at))
```
Comment (niranjan94) [inline app/helpers/ticketing.py]: @poush can you remove this line ?

**Comment by poush** [comment]
@niranjan94  Done!

File: app/helpers/ticketing.py [hunk 3]
```diff
@@ -3,7 +3,7 @@
 
 from flask import url_for, flash
 from flask.ext import login
-from sqlalchemy import asc
+from sqlalchemy import asc, desc
```
Comment (niranjan94) [inline app/helpers/ticketing.py]: Please fix codacy issues

**Comment by poush** [comment]
@niranjan94 Done again!
